docs/version0.13.md

```diff
@@ -41,10 +43,21 @@ OpenJ9 release 0.13.0 supports OpenJDK 12, which is available from the AdoptOpen
 
 OpenJDK 12 with Eclipse OpenJ9 is not a long term support (LTS) release.
 
-Although it is possible to build an OpenJDK 8, or other versions, with OpenJ9 0.13.0, testing at the project is not complete and therefore support is not available.
+The latest builds of OpenJDK with OpenJ9 for Java 8 and 11 at the AdoptOpenJDK community are for Eclipse OpenJ9 release 0.12.0. Features mentioned in these release notes are not available in these builds. Although it is possible to build an OpenJDK 8 or OpenJDK 11 with OpenJ9 0.13.0, testing at the project is not complete and therefore support for any of these features is not available.
```

Comment:
"may" be possible to build. Although it works today, a future change could break it at any time.

Reply:
Changed to "might be"